tests/integration/test_workflow.py

```diff
@@ -155,7 +155,7 @@ def assert_accuracy(self, accuracy):
 class NnftWorkflowTestCase(WorkflowTestCase):
     def setUp(self):
         super().setUp()
-        self.min_accuracy = 0.61
```

Comment:
Suggest to add some comments to explain min_accuracy, prev_accuracy and (in code above) expected_accuracy.

Reply:
I've added a couple of comments.